### ISSUE TITLE
fix for "error gathering device information while adding custom device "/dev/kvm": no such file or directory."

### ISSUE THREAD

**Issue by fluffy-git** [sickcodes/Docker-OSX#786]
not really an issue, but wanted to share this so others don't have to go through hours of troubleshooting like i had 

### tl;dr: don't use docker-desktop on linux.

Docker desktop changes docker config so that all docker commands execute through docker-desktop, and for some reason docker desktop is unable to see /dev/kvm/. 
so the solution to this problem when having docker-desktop installed, is to **completely** remove docker desktop

to completely remove docker-desktop first follow [this guide](https://docs.docker.com/desktop/uninstall/) in the docker-desktop docs.
(you might also want to delete ~/.docker, but be careful, this deletes all docker configs and containers as far as i know, so make a backup of it)

now execute these two commands to return the docker context to normal:
- `docker context ls`
- `docker context use default`

now restart and it should be fixed if everything else is configured correctly
